Snap_rounding_2/include/CGAL/Float_snap_rounding_2.h

```diff
@@ -0,0 +1,452 @@
+// Copyright (c) 2025 Geometry Factory.
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+//
+// author(s)     : Léo Valque
+
+#ifndef CGAL_FLOAT_SNAP_ROUNDING_2_H
+#define CGAL_FLOAT_SNAP_ROUNDING_2_H
+
+
```

Comment:
```suggestion

#include <CGAL/license/Snap_rounding_2.h>

```

Reply:
No, file should be move with a dedicated license. Already in the TODOs